pkgs/development/python-modules/pynvim-pp/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pynvim
+, setuptools
+}:
+
+buildPythonPackage {
+  pname = "pynvim-pp";
+  version = "unstable-2023-07-05";
+
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "ms-jpq";
+    repo = "pynvim_pp";
+    rev = "e9ac3171c7edcc0be020d63840d6af4222b69540";
+    fetchSubmodules = false;
+    hash = "sha256-H1sCwU2wl9HO92LNkvkCb1iLbZrTNMmYA/8qy9uCgyU=";
+  };
+
+  propagatedBuildInputs = [
+    pynvim
+    setuptools
```

Comment:
Is this required?

Reply:
Yes, I tried without. Adding `setuptools` to `propagatedBuildInputs` seems to be common for `format = pyproject` packages.

Reply:
works for me after moving setuptools to `nativeBuildIputs`